app/models/declaration/TransportInformation.scala

```diff
@@ -23,6 +23,8 @@ case class TransportInformation(transportPayment: Option[TransportPayment] = Non
 
   private def hasContainer(id: String) = containers.exists(_.id == id)
 
+  val hasContainers: String = if (containers.nonEmpty) "Yes" else "No"
```

Comment:
You would naturally expect a method called `hasContainers` to return a `Boolean`. The summary page can change that to a Yes/No if required

Reply:
Name is good but aggree it should return boolean and mapping to Yes/No should be on view.

Reply:
ExportDeclaration has this method already (if that helps)

Reply:
Removed. This logic is handled in the view